tests/std/tests/LWG3018_shared_ptr_function/test.cpp

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <cassert>
+#include <memory>
+
+using namespace std;
+
+int val = 0;
+
+struct decrement_val {
+    void operator()(void (*)(int)) {

```

Comment:
```suggestion
    void operator()(void (* const)(int)) {

```

Reply:
I'll commit these when test are finished to make sure they all pass. I don't want to restart them now.

Reply:
Also all of these suggestions add a new line which clang-format may not like

Reply:
Edited

Reply:
Thanks